server/helpers/jwt.js

```diff
@@ -0,0 +1,23 @@
+import jwt from "jsonwebtoken";
+
+/**
+ * @description This method accepts user object and 
+ returns the requested token for the user
+ * @param {object}  user
+ * @returns {String}  jwtToken  
+ */
+const jwtEncode = user => {
+  const { id, email, lastName, firstName, isAdmin } = user;
+  return jwt.sign(
+    {
```

Comment:
Sync with your teammates on this I do not believe you require all of these parameters in your jwt. Please remember